lib/check-element.js

```diff
@@ -4,6 +4,7 @@ module.exports = checkElement
 function checkElement (element, op) {
   if (op === 'del') {
     if (!element.changeset) {
+      // TODO: Is this true for observations?
```

Comment:
I don't think so? AFAIK we've never generated changeset on observation creation. I can't think of a good reason for enforcing this.

cc @gmaclennan

Reply:
Oh! This is for deletions. Uhhhh, good Q. I say we don't require it until we know we need it.